src/tests/api/HappyPath.test.ts

```diff
@@ -0,0 +1,244 @@
+import { sessionPost, stubStartPost, postDocumentSelection, startStubServiceAndReturnSessionId, getSessionById, getSessionAndVerifyKey, authorizationGet, tokenPost, userInfoPost, sessionConfigurationGet, postAbortSession } from "./ApiTestSteps";
```

Comment:
Looks like we're getting a lint error about function length, I normally ignore this for tests by adding `/* eslint-disable max-lines-per-function */` to the top of the file

Reply:
Added this to all 4 test files